docs/ROADMAP.md

```diff
@@ -15,18 +15,17 @@ developers manage private data more securely.
 We plan for each language implementation of Asherah to undergo periodic security audits. As we
 have progressed from testing internally to releasing the codebase as open source to gather more
 feedback about the primary use cases we wish to tackle, these plans are currently tentative.
-Right now, our Java and C# implementations have reached our 1.0.0 feature set and we plan to 
-perform a security audit for these in Q3 2019. For Golang, we plan to reach 1.0.0 in Q3
-2019 and peform an audit Q4 2019.
+Right now, our Java, Go and C# implementations have reached our 1.0.0 feature set and we plan to 
+perform a security audit for these in Q4 2020.
 
 Generally, we will target a same- or next-quarter audit as languages hit major version milestones.
 
 
 ## Languages
 
-Beyond our Java, C# and Golang releases we have already planned out releases for Python and 
-Javascript/ECMAScript. These are tentatively targeted for Q3 2019. We hope to add other 
-languages as we get interest in them.
+Beyond our Java, C# and Golang releases we have already planned out releases for a Go and JAVA based 
+sidecar. Python and Javascript/ECMAScript are tentative languages and we plan to develop native SDKs for
```

Comment:
sidercar/server is already available for use. `we have already planned out` needs to be reworded.